triveni-codev/GBWebApp-dev
# Darlp networks atsc cable virtual channel 1203486985241019

Hi @triveni-codev , @chriscodev , @pauljustinhernandez 

Kindly check this PR ONLY FOR DIRTY CHANGE and RETRIEVAL/DISPLAY FROM SERVER on both add and update functionalities ONLY FOR ATSC CABLE. 

2 ASANA Tickets associated: 
ASANA: https://app.asana.com/0/1201943683850147/1203486985241019
ASANA: https://app.asana.com/0/1201943683850147/1203486985241021

To test: 
-Virtual Channel should consider the following: 
>1 to 16383 as Major only (no Minor value) is acceptable
>999-999 as biggest Major-Minor value is acceptable
>0-(1 to 999) as Major-Minor values are acceptable
>(1 to 999)-0 as Major Minor values are acceptable
>(1 to 999)-(1 to 999) Major-Minor value combinations are acceptable
>You may now test the BIG COMMIT for both ADDING and UPDATING.

Notes:
0 is not acceptable as Major only value
0-0 is not acceptable as Major-Minor value

Please also test the PATH SELECT and OUT OF BAND checkboxes and make sure they are working.
